core/src/main/resources/hudson/model/ComputerSet/index.jelly

```diff
@@ -33,6 +33,7 @@ THE SOFTWARE.
   <l:main-panel>
     <l:app-bar title="${%Nodes}">
       <j:getStatic var="createPermission" className="hudson.model.Computer" field="CREATE"/>
+      <j:getStatic var="extendedReadPermission" className="hudson.model.Computer" field="EXTENDED_READ"/>
```

Comment:
In the wrong scope, will not be visible where it is needed. Seriously considering setting aside a week to prototype https://issues.jenkins.io/browse/JENKINS-31203 at this point!